kml/CMakeLists.txt

```diff
@@ -24,3 +24,7 @@ omim_add_library(${PROJECT_NAME} ${SRC})
 
```

Comment:
I am OK with this new tool, but I suggest implementing KMB import on iOS to solve the problem. Android already support KMB files.

Reply:
You can't access app's files on iOS without an external mac and Finder.

Reply:
But you can open KMB files in your app. Just share file to Organic Maps to convert from KMB into proper format.